test/test_cuda/test_export.py

```diff
@@ -344,7 +344,7 @@ def test_nvfp4_llmcompressor_format(self):
             dataset=self.llm_dataloader,
         )
         quantized_model_path = self.save_dir
-        compressed_model, _ = autoround.quantize_and_save(output_dir=quantized_model_path, format="llmcompressor")
+        compressed_model, _ = autoround.quantize_and_save(output_dir=quantized_model_path, format="llm_compressor")
```

Comment:
where are the uts for mxfp/nvfp/fp8_static, iters=1 and iters=0 should be covered